lms/static/scripts/frontend_apps/components/test/SubmitGradeForm-test.js

```diff
@@ -67,6 +69,14 @@ describe('SubmitGradeForm', () => {
     assert.equal(wrapper.find('input').key(), fakeStudent.LISResultSourcedId);
   });
 
+  it('clears out the previous grade when changing students', async () => {
+    const wrapper = renderForm();
+    await fakeFetchGrade.resolves();
+    assert.equal(wrapper.find('input').prop('defaultValue'), 10);
```

Comment:
Is the value `10` here because the fetch has resolved and then it's been multiplied by 10?